### PR TITLE
os/board/rtl8721csm: add DEBUG_SET_CALLER_ADDR() for malloc

### PR DESCRIPTION
**Pull request by ziyik**

### CODE REVIEW

File: os/board/rtl8720e/src/component/bluetooth/os/tizenrt/osif_tizenrt.c [hunk 1]
```diff
@@ -617,7 +617,7 @@ bool osif_msg_peek(void *p_handle, void *p_msg, uint32_t wait_ms)
 /****************************************************************************/
 void *osif_mem_alloc(RAM_TYPE ram_type, size_t size)
 {
-	return kmm_malloc(size);
+	return rtw_malloc(size);
```
Comment (ewoodev) [inline os/board/rtl8720e/src/component/bluetooth/os/tizenrt/osif_tizenrt.c:620]: Is this function wrapping rtw_malloc()?
If re-wrapping the warpped function of malloc API, we need to add DEBUG_SET_CALLER_ADDR() macro.

This function(osif_mem_alloc()) seems to be called from many location.
so I think, we need this for tracking who is actually asking alloc.

Comment (ziyik) [reply os/board/rtl8720e/src/component/bluetooth/os/tizenrt/osif_tizenrt.c]: You are right, I have modified it.

**Review by ewoodev — COMMENTED** [review]
```
void *osif_mem_aligned_alloc(RAM_TYPE ram_type, size_t size, uint8_t alignment)
{
	return kmm_memalign(alignment, size);
}
```

This function has alloc operation too. 
Please add DEBUG_SET_CALLER_ADDR() when succes alloc.

**Review by ewoodev — COMMENTED** [review]
os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c 
132, 137 line
```
u8 *_tizenrt_malloc(u32 sz)
{
	return kmm_malloc(sz);
}
u8 *_tizenrt_zmalloc(u32 sz)
{
	return kmm_zalloc(sz);
}
```

This line check please.

**Comment by ziyik** [comment]
> Check please below lines.
> 
> os/board/rtl8721csm/src/component/os/tizenrt/tizenrt_service.c 132, 137 line
> 
> ```
> u8 *_tizenrt_malloc(u32 sz)
> {
> 	return kmm_malloc(sz);
> }
> u8 *_tizenrt_zmalloc(u32 sz)
> {
> 	return kmm_zalloc(sz);
> }
> ```

_tizenrt_malloc and _tizenrt_zmalloc are called by rtw_malloc and rtw_zmalloc, and DEBUG_SET_CALLER_ADDR() has been added.

**Review by sunghan-chang — APPROVED** [review]

**Pull request merged**